segmentation/utils/spatial_analysis_utils.py

```diff
@@ -41,41 +41,41 @@ def calc_dist_matrix(label_map, ret=True, path=None):
         np.savez(path + "dist_matrices.npz", **dist_matrices)
 
 
-def get_pos_cell_labels(analysis_type, pheno=None, fov_data=None,
-                        thresh=None, fov_channel_data=None, cell_labels=None, col=None):
+def get_pos_cell_labels(analysis_type, pheno=None, current_fov_data=None,
+                        thresh=None, current_fov_channel_data=None, cell_labels=None, specific_marker=None):
```

Comment:
The docstring descriptoin for this function needs to be updated, doesn't really make sense as written